### PR TITLE
Adding documentation about loadbalancer-ips config option.

### PR DESCRIPTION
**Pull request by hyperbolic2346**
As requested in https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/5

### CODE REVIEW

File: pages/k8s/custom-loadbalancer.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+ -  If a full load balancing solution is in place such as an F5 appliance, remove the
+     `kubeapi-load-balancer` and use the settings on the `kubernetes-master` charm to
+      configure the load balancer.
+
+Both solutions are described in the sections below.
+
+# Virtual IP in front of kubeapi-load-balancer
+
+If you have a custom setup with a virtual IP in front of the kubeapi-load-balancer
```
Comment (hyperbolic2346) [inline pages/k8s/custom-loadbalancer.md]: do we need "custom setup" here? We don't want to define how you do it, but as long as you have a virtual IP in front of the load balancers you are good.

File: pages/k8s/custom-loadbalancer.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+Then configure the IP addresses provided by the load balancing solution with the
+`kubernetes-master` charm.
+
+```bash
+juju config kubernetes-master loadbalancer-ips="192.168.1.1 192.168.2.1"
+```
+
+Multiple IP addresses should be given as a space-separated list.
```
Comment (hyperbolic2346) [inline pages/k8s/custom-loadbalancer.md]: space separated? :rofl:

Comment (evilnick) [reply pages/k8s/custom-loadbalancer.md]: ha ha!
Although this is actually annoying - why some things are comma-separated and others space-separated. anyhow

File: pages/k8s/custom-loadbalancer.md (hunk 1 above)
Comment (evilnick) [reply pages/k8s/custom-loadbalancer.md]: Ok, I will reword this.

**Comment by evilnick** [comment]
does anything from this also need to be mentioned in #167 ?

**Comment by evilnick** [comment]
:carrot:

**Pull request merged**